cellpack/autopack/upy/dejavuTk/dejavuHelper.py

```diff
@@ -19,15 +18,13 @@
     along with upy.  If not, see <http://www.gnu.org/licenses/gpl-3.0.html>.
 """
 
-# standard module
+# standardmodule
 import os
 import numpy
 from PIL import Image
-import collada
-
-from cellpack.autopack.upy import hostHelper
 
 # DejaVu module
+from cellpack.autopack.upy import hostHelper  # its confusing that upy is in three different place
```

Comment:
I'm trying to delete mgl tools, so I'm copying any part of upy that I need into thee autopack module. Is that what you mean about 3 different places?